lib/common/common/src/mmap_hashmap.rs

```diff
@@ -4,7 +4,7 @@ use std::fs::File;
 use std::hash::Hash;
 use std::io::{self, Cursor, Write};
 use std::marker::PhantomData;
-use std::mem::size_of;
+use std::mem::{align_of, size_of};
```

Comment:
Nit: starting from Rust 1.80, we don't need to import both of these.
https://releases.rs/docs/1.80.0/#libraries

I've pushed the fix.